src/libraries/System.Console/ref/System.Console.cs

```diff
@@ -8,51 +8,112 @@ namespace System
 {
     public static partial class Console
     {
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("android")]
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static System.ConsoleColor BackgroundColor { get { throw null; } set { } }
-        public static int BufferHeight { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
-        public static int BufferWidth { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
+        public static int BufferHeight 
+        { 
```

Comment:
These files are generated by a tool. We can't make manual changes to it because they will get undone when someone regenerates the file.

See [https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md)

Reply:
Fixed, thanks